src/pages/UserSettings/UserSettings.tsx

```diff
@@ -191,35 +194,6 @@ export default function UserSettings() {
         )}
 
         <h4 className="text-gray-600">Account Settings</h4>
-        <TextField
-          value={email}
-          hint={user!.email}
-          onChangeValue={setEmail}
-          label="Change email:"
-          type="email"
-        />
-        {changeEmailState && (
-          <label
-            className={`font-normal ${
-              emailError || changeEmailState === "Email not valid"
-                ? "text-error"
-                : "text-gray-400"
-            }`}
-          >
-            {changeEmailState}
-          </label>
-        )}
-        <Button
-          label="Change Email"
-          icon={emailChangeLoading ? <Spinner variaton="SMALL" /> : undefined}
-          onClick={() => {
-            if (EmailValidator.validate(email)) {
-              changeEmail(email);
-            } else {
-              setChangeEmailState("Email not valid");
-            }
-          }}
-        />
         <TextField
```

Comment:
same comment about the supportingText/errorText props (i think this would be better since the component would automatically set the color of the error text and would give the text field a red ring)